Composer/packages/lib/code-editor/src/lg/constants.ts

```diff
@@ -22,7 +22,7 @@ export type LgCardTemplateType = typeof lgCardAttachmentTemplates[number];
 export const cardTemplates: Record<LgCardTemplateType, string> = {
   adaptive: `> To learn more Adaptive Cards format, read the documentation at
 > https://docs.microsoft.com/en-us/adaptive-cards/getting-started/bots
-- \${{
+- \`\`\`\${json({
```

Comment:
we don't want the use of `json` here, the following should work:
```
- ```{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "TextBlock",
      "text": "default text",
      "weight": "bolder",
      "isSubtle": false
    }
  ]
}```
```

Reply:
Oh, I see. So we still want the Activity template to use`${json(template_name()}`, but we don't want to use json in the adaptive card template string.

Reply:
Yes, but the following also works in PVA:

```
# SendActivity_yhVIg9()
[Activity
    Attachments = ${SendActivity_yhVIg9_attachment_h83mGt()}
]

# SendActivity_yhVIg9_attachment_h83mGt()
> To learn more Adaptive Cards format, read the documentation at
> https://docs.microsoft.com/en-us/adaptive-cards/getting-started/bots
- ```{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "TextBlock",
      "text": "default text",
      "weight": "bolder",
      "isSubtle": false
    }
  ]
}```
```

Reply:
Yes TJ exactly

Reply:
When I exclude the `json()` all together, it doesn't work in the Web Chat panel.
```
# SendActivity_yhVIg9()
[Activity
    Attachments = ${SendActivity_yhVIg9_attachment_h83mGt()}
]

# SendActivity_yhVIg9_attachment_h83mGt()
> To learn more Adaptive Cards format, read the documentation at
> https://docs.microsoft.com/en-us/adaptive-cards/getting-started/bots
- ```{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "TextBlock",
      "text": "default text",
      "weight": "bolder",
      "isSubtle": false
    }
  ]
}```
```

Reply:
That's right I tested the same thing myself (edited my previous message)